/**:
  ros__parameters:
    port:
      name: "/dev/ttyUSB0"
      baudrate: 115200
    topic:
      output_hz: 10
      topic: "serial_data"
    tf:
      enable: true
      base_frame: "base_link"
      node_frame: "node_link"